channels/sa.m3u

```diff
@@ -1,11 +1,7 @@
 #EXTM3U x-tvg-url="https://raw.githubusercontent.com/Fazzani/grab/master/merge.zip"
-#EXTINF:-1 tvg-id="makkah.sa" tvg-name="-makkah" tvg-country="SA" tvg-language="" tvg-logo="" group-title="",-makkah
-http://mbningestworld-i.akamaihd.net/hls/live/644021/iraqworldsafe/master_2596.m3u8
```

Comment:
Working link. Not duplicate.

Reply:
It's Al Hurra Iraq which doesn't belong in sa.m3u. It's already in iq.m3u

https://github.com/iptv-org/iptv/blob/86fbb63ea771d3445220235c42ea842bf33952d2/channels/iq.m3u#L7